background/ncn_boot_workflow.md

```diff
@@ -15,6 +17,27 @@ Non-compute nodes boot two ways:
 * [Reverting Changes](#reverting-changes)
 * [Locating USB Device](#locating-usb-device)
 
+<a name="determine-the-current-boot-order"></a>
+
+### Determine the current bootorder
+
+Under normal operations, the NCNs use the following boot order:
+
+1. PXE (to allow for booting with changes in BSS)
+2. Disk (fallback in the even PXE services are unavailable)
+
+<a name="reasons-to-change-the-bootorder"></a>
+
+#### Reasons to change the boot order
+
+Under normal operations, it is not necessary to change the boot order.  Having PXE first and disk as a fallback works in the majority of situations.
+
+It may be desireable to change this order if:
+
+- you need to test disk-backed booting
```

Comment:
`s/you/one/g` 

Less direct, and more generic.

Reply:
I mean to comment this throughout, but my regex /g flag was an attempt to say that :)